reference/requirements.rst

```diff
@@ -3,19 +3,44 @@
 
 .. _requirements-for-running-symfony2:
 
-Requirements for Running Symfony
-================================
+Requirements for Running Symfony 4
+==================================
 
 Symfony 4 requires **PHP 7.1.3** or higher to run, in addition to other minor
-requirements. To make things simple, Symfony provides a tool to quickly check if
-your system meets all those requirements. Run this command to install the tool:
+requirements:
+
+PHP Extensions
+~~~~~~~~~~~~~~
+* The `Ctype`_ extension must be available
+* The `iconv`_ extension  must be available
+* The `JSON`_ extension must be available
+* The `PCRE`_ extension must be available (minimum version 8.0)
+* The `Session`_ extension must be available
+* The `SimpleXML`_ extension must be available
+* The `Tokenizer`_ extension must be available
+
+Please note that all these extensions are installed and enabled by default 
+in PHP 7+.
+
+Other Requirements
+~~~~~~~~~~~~~~~~~~
+* The cache directory must me writable by the web server
+* The logs directory must be writable by the web server
```

Comment:
Is it the case when installing the default skeleton? By default logs are outputted to `stderr` so it looks like an arbitrary limitation (especially annoying in a containerized environment).

Reply:
yes, it's the case, that's a feature provided by the kernel: ensuring there is a writeable directory available for logs (and cache) - unrelated to the actual logger that is used (or not)

Reply:
One known issue is that the `debug-pack` installs Monolog... then the logs aren't outputted to stderr anymore (Monolog is a transient deps of `EasyCorp/easy-log-handler`,  that is part of this pack). But in prod, no logs directory should be necessary.